### PR TITLE
robotProblem

### PR DESCRIPTION
**Pull request by aaspeel**
upload urdf of the robot + code to instantiate the system

### CODE REVIEW

File: problems/biped_robot.urdf [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      <geometry>
+         <cylinder length="1.1" radius=".03" />
+      </geometry>
+      <material name="blue">
+	       <color rgba="0 0 1 1" />
+      </material>
+    </visual>
+  </link> -->
+
+</robot>
```
Comment (egidioln) [inline problems/biped_robot.urdf:139]:
```suggestion
</robot>

```

Comment (egidioln) [reply problems/biped_robot.urdf:139]: I think the urdf file should be placed I another folder (maybe `\deps`?).

File: problems/robot_problem.jl [hunk 2]
```diff
@@ -0,0 +1,69 @@
+using RigidBodyDynamics
```
Comment (blegat) [inline problems/robot_problem.jl:1]: Create a module like other problems

File: problems/robot_problem.jl [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+using MathematicalSystems
+using Dionysos
+
+# #### If we want to load the URDF instead of creating the robot with RigidBodyDynamics
+## describe path to urdf file
+#packagepath() = joinpath(@__DIR__, "..","..", "deps")
+urdfpath() = joinpath(@__DIR__, "biped_robot.urdf")
+
+## load mechanism 'robot' from the URDF file
+robot = parse_urdf(Float64, urdfpath())
```
Comment (blegat) [inline problems/robot_problem.jl:14]: Put everything in a `system` function

**Comment by ThiagoAlvesLima** [comment]
Hi, I created a new pull request with the required updates and some improvements by using the new Biped Robot package in the Robot problem code. I think we can then close this.

**Comment by blegat** [comment]
Thanks, I modified the comment of your PR so that this PR is closed automatically when your PR is merged

**Pull request closed without merging**